tt_metal/hw/ckernels/wormhole_b0/common/inc/ckernel.h

```diff
@@ -36,7 +36,7 @@
 #define OVERLAY_DECOUPLE 0
 #endif
 
-#ifdef LLK_TB_TEST
+#if defined(EN_KERNEL_SLOWDOWN)
```

Comment:
Hi @muthutt , its because the rest of the defines all use `#ifndef`, so changing the `#ifdef` to `#if defined` provides a more visible difference than using `ifdef` and `ifndef` sequentially in the same file (just easier for debugging). If its an issue though for the coding standard I can definitely change it.

Reply:
Hey @muthutt , since the above file is common between buda & metal and will be in its own repo soon, we will keep that define to keep it parity until the common submodule